### PR TITLE
Multi-Node local devnet

### PR DESCRIPTION
**Pull request by MiguelDD1**
# Multi-Node Local DevNet

This new `chain-spec` defines a network using 2 validators: `Alice` and `Bob` and the docker-compose file to deploy them easily into your local. 

    $> docker-compose -f docker-compose.dev.yml up

## Details
1. `Alice` and `Bob` are invulnerable.
2. RPC port exported port is 9944, so you can use `http://polygon-da-explorer.matic.today/#/explorer` and switch to local node to connect with your nodes.

## Prune your network

The docker compose uses volumes to store states of the network. You can prune your state using:

    $ docker-compose -f docker-compose.dev.yml down -v

### CODE REVIEW

File: docker-compose.dev.yml [hunk 1]
```diff
@@ -6,18 +6,13 @@ services:
       context: ./images/da
       args:
         - BRANCH=miguel/mat-3347-new-chain-spec-for-dev-testnet
-    image: da:latest
+    image: da:mat-3347
```
Comment (kroos47) [inline docker-compose.dev.yml:9]: is the image mat-3347 pushed to docker hub?

Comment (kroos47) [inline docker-compose.dev.yml:9]: is the image mat-3347 pushed to docker hub?

Comment (sh3ll3x3c) [reply docker-compose.dev.yml:9]: @kroos47 or we could just create the image with `mat-3347` tag locally

**Review by sh3ll3x3c — APPROVED** [review]

File: docker-compose.dev.yml (hunk 1 above)
Comment (kroos47) [reply docker-compose.dev.yml:9]: yeh I'm doing with that!

**Review by kroos47 — APPROVED** [review]

File: docker-compose.dev.yml (hunk 1 above)
Comment (MiguelDD1) [reply docker-compose.dev.yml:9]: No,  the `da:mat-3347` was not pushed. However it is automatically built by docker-compose locally (see `build` at line 5).
You can also build it manually by:

    $> docker-compose -f docker-compose.dev.yml build alice

Now that `avail` is public the construction of the docker image has been simplified.

**Pull request merged**